composer.json

```diff
@@ -16,18 +16,29 @@
     "ext-mbstring": "*",
     "ext-xsl": "*",
     "monolog/monolog": "1.27.*",
-    "symfony/translation": "6.0.*",
+    "symfony/translation": "6.4.*",
     "symfony/translation-contracts": "3.0.*",
-    "symfony/twig-bundle": "6.0.*",
-    "symfony/config": "6.0.*"
+    "symfony/twig-bundle": "6.4.*",
+    "symfony/config": "6.4.*",
```

Comment:
The new symfony dependencies are only needed to get the console working, so they could perhaps be moved into dev dependencies